clang/bindings/python/clang/cindex.py

```diff
@@ -1820,6 +1820,18 @@ def availability(self):
 
         return AvailabilityKind.from_id(self._availability)
 
+    @property
+    def binary_operator(self):
+        """
+        Retrieves the opcode if this cursor points to a binary operator
+        :return:
+        """
+
+        if not hasattr(self, "_binopcode"):
+            self._binopcode = conf.lib.clang_Cursor_getBinaryOpcode(self)
```

Comment:
Have you checked that the tests pass? I am not completely sure about the inner workings of this, but it seems  to me that all `conf.lib` functions should be registered with the lib object by adding an entry for the function in the `functionlist` object at the end of the file.
For comparison, see e.g. `clang_Cursor_getOffsetOfField`

Reply:
Good point, you are probably right, thank you! There is also an `__all__` list at the bottom of `cindex.py` listing, among others, all enumeration classes defined in `cindex.py`. Probably the new `BinaryOperator` class should be added there as well? I'm currently running the tests locally.

Reply:
You're right, I missed that - that's also necessary so you get `BinaryOperator` when doing `from cindex import *`

Reply:
Thanks for your review and explanation! I've just pushed a new version, addressing both issues.